modules/static/prometheus/client-model/rsync.incl

```diff
@@ -0,0 +1,4 @@
++ LICENSE
++ *.proto
```

Comment:
All proto files from the repo. It has just one: https://github.com/prometheus/client_model/blob/master/io/prometheus/client/metrics.proto